build.ps1

```diff
@@ -188,7 +188,14 @@ $skip_test_projects_on_windows = @("tree-sitter-dscexpression")
 
             if (Test-Path "./copy_files.txt") {
                 Get-Content "./copy_files.txt" | ForEach-Object {
-                    Copy-Item $_ $target -Force -ErrorAction Ignore
+                    # copy the file to the target directory, creating the directory path if needed
```

Comment:
throw an error if a backslash is used as separator so that they are all consistently using forward slash

Reply:
added